app/src/main/java/com/github/se/cyrcle/ui/parkingDetails/ParkingDetailsAlertDialogShowImage.kt.kt

```diff
@@ -0,0 +1,60 @@
+package com.github.se.cyrcle.ui.parkingDetails
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.ArrowBack
+import androidx.compose.material3.BasicAlertDialog
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import coil.compose.rememberAsyncImagePainter
+import com.github.se.cyrcle.ui.theme.atoms.IconButton
+
+/**
+ * Alert dialog to show the image of a parking spot. This composable is displayed when the user
+ * clicks on an image of a parking spot.
+ *
+ * @param onDismiss Callback when the dialog is dismissed.
+ * @param imageUrl URL of the image to display.
+ */
+@Composable
+@OptIn(ExperimentalMaterial3Api::class)
+fun ParkingDetailsAlertDialogShowImage(onDismiss: () -> Unit, imageUrl: String) {
```

Comment:
I'm confused, does this come from @sean-prz's PR? Did you rebase on main correctly?

Reply:
Just noticed there was an issue with that... didn't really do anything different to normal. Oh well, unfortunately nothing can really be done about it now and it's adding the same code onto main so I'll leave it as is